src/mpi/init/init_global.c

```diff
@@ -158,6 +158,8 @@ int MPII_post_init_global(int thread_provided)
     /* Assert: tag_ub is at least the minimum asked for in the MPI spec */
     MPIR_Assert(MPIR_Process.attrs.tag_ub >= 32767);
 
+    MPII_Timer_init(MPIR_Process.comm_world->rank, MPIR_Process.comm_world->local_size);
```

Comment:
my only comment is why move this here and not just call directly in `MPIR_Init_thread`. I agree it was out of place in the memory tracing init function.

Reply:
It seems `MPII_Timer_init` is only needed for `rlog` and it is really part of rlog init. Since its purpose (or components) is not clear at the top level in `MPIR_Init_thread`, I find it out-of-place there. Right now it is in `init_global`, a misc place that waits for further refactoring.

Reply:
fair enough. really only a curiousity. since rlog is still broken it's probably not worth nitpicking over it.